vespalib/src/vespa/vespalib/fuzzy/fuzzy.cpp

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
Since this is original code based on apache code, you should replace this entire header by these two lines:

// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
// Levenstein distance algorithm is based off Java implementation from apache commons-text library licensed under the Apache 2.0 license.